python/nvstrings/cpp/pytext.cpp

```diff
@@ -649,6 +649,50 @@ static PyObject* n_scatter_count( PyObject* self, PyObject* args )
     return PyLong_FromVoidPtr((void*)strs);
 }
 
+//
+static PyObject* n_porter_stemmer_measure( PyObject* self, PyObject* args )
+{
+
+    PyObject* pystrs = PyTuple_GetItem(args,0);
+    NVStrings* strs = strings_from_object(pystrs);
+
+     if( strs==0 )
+        Py_RETURN_NONE;
+
+    const char * vowels = "aeiou" ;
```

Comment:
Are you sure you want to hardcode these?  Perhaps make them defaults inside the python method?

Reply:
Changed to default-python arguments.

 I dont think it should matter though as this is an internal functions to the algorithm. 

https://github.com/rapidsai/cudf/blob/9ffa721fb37a63b03513ba84e8256ecd7f547f89/python/nvstrings/nvtext.py#L351


https://github.com/rapidsai/cudf/blob/170e59be2442c079e56a950799c4c5e954712f33/python/nvstrings/cpp/pytext.cpp#L662-L670